src/modules/menu/components/activeLayers/MenuActiveLayersListItemTimeSelector.vue

```diff
@@ -150,4 +153,8 @@ function hidePopover() {
         white-space: nowrap;
     }
 }
+
+.w-13 {
+    width: 13%;
+}
```

Comment:
you could move that to the bootstrap theme SCSS file, so that it will be available to the whole app